doc/user/zebra.rst

```diff
@@ -358,6 +370,139 @@ default) should the specified gateways not be reachable. E.g.:
 
 .. _multicast-rib-commands:
 
+:abbr:`VRF (Virtual Routing and Forwarding)`
+======================================
```

Comment:
Could you extend the underline out to the end of the heading? Also, I have been following the convention of not using abbreviations in section titles; I think it's a good idea here.